platforms/openal/SoundSystemAL.cpp

```diff
@@ -5,77 +5,16 @@
 
 SoundSystemAL::SoundSystemAL()
 {
-	loaded = false;
-	
-	device = alcOpenDevice(NULL);
-	if (!device)
-	{
-		LOG_E("Unable To Load Audio Engine");
-		return;
-	}
-
-	// Create Context
-	context = alcCreateContext(device, NULL);
-	ALCenum err = alcGetError(device);
-	if (err != ALC_NO_ERROR)
-	{
-		LOG_E("Unable To Open Audio Context: %s", alcGetString(device, err));
-		return;
-	}
-
-	// Select Context
-	alcMakeContextCurrent(context);
-	err = alcGetError(device);
-	if (err != ALC_NO_ERROR)
-	{
-		LOG_E("Unable To Select Audio Context: %s", alcGetString(device, err));
-		return;
-	}
-
-	// Set Distance Model
-	alDistanceModel(AL_LINEAR_DISTANCE_CLAMPED);
-
-	// Mark As Loaded
-	loaded = true;
+	_initialized = false;
+    _listenerVolume = 1.0;
+    _audioMuted = false;
+    
+    startEngine();
```

Comment:
Also indentation inconsistency, maybe we should add clang-format file.

Reply:
I've had bad experiences with clang format. Maybe someone should submit a pull request that adds it and that also doesn't mess with the code base itself too much?